packages/origin-247-certificate/src/evented-certificate/repositories/CertificateEvent/CertificateEvent.repository.ts

```diff
@@ -22,6 +22,7 @@ export type NewCertificateEvent = Omit<CertificateEventEntity, 'id' | 'createdAt
 
 export interface CertificateEventRepository {
     create(certificateEvent: NewCertificateEvent): Promise<CertificateEventEntity>;
```

Comment:
This method is not used anymore, right?

Reply:
It's not used but I've left it

Reply:
I wouldn't mind leaving it, but I've deleted